interview-questions.md

```diff
@@ -8,44 +8,72 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
-
-Researched answer:
+Your answer: Super is used when creating a child class and it is used to reference an attribute that is being inherited from the parent class.
+
+For example:
+```rb
+# create parent class
+class Car
+    def initialize (color, transmission, engine_size)
+        @color = color
+        @transmission = transmission
+        @engine_size = engine_size
+    end
+end
+
+# create child class 
+class Supercar < Car
+    def initialize (color, transmission, engine_size, max_speed)
+        super(color)
+        super(transmission)
+        super(engine_size)
+        @max_speed = max_speed
```

Comment:
You can use one `super()` call to have access to all of the attributes, no need to make 3 separate `super()` calls. Great job giving an example though!

Reply:
oh thank you! I was trying to figure that out and I did a separate test_file.rb but couldn't make it work. It kept giving me weird errors, and I think it is because I was calling super 3 times